wazo_confd/plugins/event_handlers/plugin.py

```diff
@@ -38,47 +32,16 @@ def _auth_tenant_added(self, event):
             self.service.copy_slug(tenant, slug)
 
     def _auth_tenant_deleted(self, event):
-        tenant_uuid = event['uuid']
-        logger.debug('Deleting devices for tenant "%s"', tenant_uuid)
-        devices = self.provd.devices.list(tenant_uuid=tenant_uuid, recurse=True)[
-            'devices'
-        ]
-        for device in devices:
-            self._delete_device(device)
-            self._delete_config(device['config'])
-
-    def _delete_device(self, device):
-        device_model = Device(device)
-        self.provd.devices.delete(device_model.id)
-        self.device_notifier.deleted(device_model)
-
-    def _delete_config(self, config):
-        try:
-            self.provd.configs.delete(config)
-        except ProvdError as e:
-            if e.status_code != 404:
-                raise
-        except Exception:
-            pass  # device has no config
+        remove_tenant(event['uuid'])
```

Comment:
Where does the provd devices deletion happen now?

Reply:
Devices deletion in provd didn't work: the tenant is deleted, so the request to provd is denied (invalid token)

Reply:
Makes sense